ui/src/plugins/epl-ets/components/ETS.js

```diff
@@ -121,6 +227,9 @@ class ETS extends Component {
                   tabs: this.props.data.etsNodeTabs,
                   node: this.state.clickedNode
                 }}
+                tableClicked={tabIds => {
```

Comment:
You can just pass function you're calling at the end:
`tableClicked={this.handleTabClick}`